packages/table/src/Table.tsx

```diff
@@ -1,19 +1,84 @@
 import React from 'react';
+import debounce from 'lodash/debounce';
+import { transparentize } from 'polished';
 import { cx, css } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
+import {
+  useIsomorphicLayoutEffect,
+  useViewportSize,
+} from '@leafygreen-ui/hooks';
 import { HeaderRowProps } from './HeaderRow';
 import { TableHeaderProps } from './TableHeader';
 import { TableProvider } from './TableContext';
 import TableHead from './TableHead';
 import TableBody from './TableBody';
 import { SortProvider } from './SortContext';
 
+const shadowColor = transparentize(0.6, uiColors.black);
+
+const containerStyle = css`
+  position: relative;
+  max-width: calc(100% - 100px);
+`;
+
 const tableStyles = css`
   border-collapse: collapse;
   box-sizing: border-box;
   border-bottom: 1px solid ${uiColors.gray.light2};
 `;
 
+const shadow = css`
```

Comment:
We need a `pointer-events: none` rule in these styles to ensure that the shadows don't block user interaction with elements underneath the shadow.

Reply:
👍